static/scss/style.scss

```diff
@@ -1181,6 +1181,10 @@ img.cand-pic {
   zoom: 1;
 }
 
+.u-hide {
+  display: none !important;
+}
+
```

Comment:
there's already a .hidden class -- is there any reason we need to add this one?

Reply:
I didn't realize we had .hidden. I'll use that instead.